build/make_bundle.sh

```diff
@@ -151,6 +138,4 @@ yq w $OLM_CATALOG/splunk/$VERSION/splunk.v${VERSION}.clusterserviceversion.yaml
 yq w $OLM_CATALOG/splunk/splunk.package.yaml packageName "splunk-certified" > $OLM_CERTIFIED/splunk/splunk.package.yaml
 
 # Mac OS expects sed -i '', Linux expects sed -i''. To workaround this, using .bak
-sed -i.bak "s,$DOCKER_IO_PATH/spark,$REDHAT_REGISTRY_PATH/spark,g" $OLM_CERTIFIED/splunk/splunk.v${VERSION}.clusterserviceversion.yaml
```

Comment:
Are we sure this has no other side effects on **$OLM_CERTIFIED/splunk/splunk.v${VERSION}.clusterserviceversion.yaml**?

Reply:
Good point. Do you think I should run any specific test?
Maybe updating a CRD version and running it again?

Reply:
Running a **"make generate"** and comparing should be good.

Reply:
Resolved.